schemas/com.snowplowanalytics.snowplow.media/session/jsonschema/1-0-0

```diff
@@ -0,0 +1,138 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity for media player events that tracks a session of a single media player usage",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.media",
+    "name": "session",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "mediaSessionId": {
+      "type": "string",
+      "maxLength": 256,
+      "description": "An identifier for the media session."
+    },
+    "startedAt": {
+      "type": [
+        "null",
+        "string"
+      ],
+      "description": "Date-time timestamp of when the session started.",
```

Comment:
utc or local time?

Reply:
I have used local time in the trackers – clarified it here as well.